mor/mor/feel/feelmor/parameterspace.hpp

```diff
@@ -220,8 +220,8 @@ class ParameterSpace: public CommObject, public std::enable_shared_from_this<Par
                     this->operator()( it - paramNames.begin() ) = value;
                 else
                 {
-                    Feel::cout << tc::red << "Warning: " << tc::reset << value << " not in range [" << min_val << ", " << max_val << "]" << std::endl;
-                    LOG( WARNING ) << value << " not in range [" << min_val << ", " << max_val << "]" << std::endl;
+                    LOG( WARNING ) << value << " not in range [" << min_val << ", " << max_val << "] for parameter named " << name << std::endl;
+                    throw std::invalid_argument("Parameter out of range");
```

Comment:
please use a similar message as the line above using fmt::format to produce the string